roughpy/compute/_src/fnv1a_hash.h

```diff
@@ -0,0 +1,40 @@
+#ifndef ROUGHPY_COMPUTE__SRC_FNV1A_HASH_H
+#define ROUGHPY_COMPUTE__SRC_FNV1A_HASH_H
+
+
+#include <stdint.h>
+#include <stddef.h>
+
+/*
+ * This is an implementation of the FNV-1a hash function which essentially
+ * follows the implementation here: https://github.com/lcn2/fnv. This is the
+ * same hash function that is used in NumPy (and is referenced is implemented
+ * in the Python source code).
+ */
+
+#define FNV1A_OFFSET_BASIS_64 0xCBF29CE484222325
+#define FNV1A_PRIME_64 0x00000100000001B3
```

Comment:
FNV1A_PRIME_64 (and 32) not used. See comment in `fnv1a_hash.c`

Reply:
The primes are not used in the current implementation, but there is no harm keeping the around.